### PR TITLE
#12 Add 2 players + add quarters delimitation

### PR DESCRIPTION
**Pull request by Clebiez**
# 12 I want to switch players

- [x] Add in the gameService the currentPlayer field
- [x] When rotating a quarter, switching to the other player (your turn is finished)
- [x] When adding a marble, now adding in the board the value of the player (1 or 2) 
- [x] Add some basic CSS to detect more easily quarters (borders variations)

2players

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
-html, body {
-    margin: 0;
-    padding: 0;
-    font-family: 'Arial';
-    color: white;
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Arial";
```
Comment (zyhou) [inline public/css/main.css:11]: comic sans ms 
image

File: src/Service/BoardService.php [hunk 2]
```diff
@@ -38,9 +38,9 @@ public function isPositionAvailable(array $board, array $position): bool
         return true;
     }
 
-    public function addMarble($board, $position): array
+    public function addMarble(array $board, array $position, int $value): array
     {
-        $board[$position[0]][$position[1]] = 1;
+        $board[$position[0]][$position[1]] = $value;
```
Comment (zyhou) [inline src/Service/BoardService.php:44]: I didn't see this before, can you clone/copy the array ? Immutability is powerful tools

File: templates/game/index.html.twig [hunk 3]
```diff
@@ -1,83 +1,86 @@
...
+        {% else %}
+            <h2 class="player-color--2">Player 2, this is your turn !</h2>
+        {% endif %}
 
-    {% if turnStatus == 'add_marble' %}
-    <h3>Please add a marble</h3>
-    {% else %}
-    <h3>Please turn a quarter</h3>
-    {% endif %}
+		{% if turnStatus == constant('App\\Service\\GameService::ADD_MARBLE_STATUS') %}
```
Comment (zyhou) [inline templates/game/index.html.twig:21]: +1

**Review by zyhou — CHANGES_REQUESTED** [review]

**Comment by fzaninotto** [comment]
I think you can improve the UI a bit:

- when the buttons appear, the grid moves down. It's disturbing
- why ask the user to click on buttons representing the quadrants rather than the quadrants themselves?

**Review by zyhou — APPROVED** [review]

**Pull request merged**